tests/phpunit/tests/webfonts-api/providers/wpWebfontsLocalProvider.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/**
+ * @group  webfonts
+ * @covers WP_Webfonts_Google_Provider
```

Comment:
```suggestion
 * @covers WP_Webfonts_Local_Provider
```
This should be the local provider no?

Reply:
You're right! Thank you @costdev - fixed in https://github.com/WordPress/wordpress-develop/pull/1736/commits/92bd91195f7896404e666f8a091b7224903157e9.